datahub-web-react/src/app/shared/SelectedOwnerTag.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Tag } from 'antd';
+
+type Props = {
+    closable: boolean;
+    onClose: (event?: React.MouseEvent<HTMLElement, MouseEvent> | undefined) => void;
+    label: React.ReactNode;
+};
+
+export default function SelectedOwnerTag({ closable, onClose, label }: Props) {
```

Comment:
OwnerPill

Reply:
Created a new component OwnerPill and returned the JSX as per requirement.